setup.py

```diff
@@ -29,12 +29,20 @@ def package_data(pkg, roots):
     ],
     install_requires=[
         'XBlock',
-        'xblock-utils',
-        'edx-rest-api-client',
-        'reportlab',
-        'xblock-problem-builder',
-        'six',
+        'xblock-utils==1.2.2',
+        'edx-rest-api-client==1.6.0',
+        'reportlab==3.1.44',
```

Comment:
We should not lock these since juniper is at a later version, and this will break that. 

Just lock reportlab since that is a dependency of only this block, and not of the platform.

Reply:
unpinned all except ``reportlab``